ESSArch_Core/storage/tests/test_models.py

```diff
@@ -0,0 +1,487 @@
+import os
+import shutil
+import tempfile
+import uuid
+from unittest import TestCase, mock
+
+from ESSArch_Core.configuration.models import Parameter
+from ESSArch_Core.ip.models import InformationPackage
+from ESSArch_Core.storage.models import (
+    DISK,
+    TAPE,
+    CAS,
+    get_storage_type_from_medium_type,
+    medium_type_CHOICES,
+    StorageTarget,
+    StorageMedium, TapeSlot, Robot, StorageObject)
+from ESSArch_Core.util import normalize_path
+
+
+class GetStorageTypeFromMediumTypeTests(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.medium_choices = [e[0] for e in medium_type_CHOICES]
```

Comment:
Add `super().setUpClass()` to fix errors occuring after the change to `django.test.TestCase`

```suggestion
        super().setUpClass()
        cls.medium_choices = [e[0] for e in medium_type_CHOICES]
```